src/styles/_catalog.scss

```
@@ -1,10 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
   margin: 0 auto;
 
+  @media (min-width: 1024px) {
+    & {
+      grid-template-areas: "card card card card";
+    }
+  }
```

1. You can skip the selector if you nest media queries.
2. Choose one of the cases (1 or 4) as default.
3. Write media queries with only 1 condition.

```suggestion
  @media (min-width: 1024px) {
    grid-template-areas: "card card card card";
  }
```